pkg/commands/plan/plan.go

```diff
@@ -257,6 +256,33 @@ func (c *PlanCommand) Run(ctx context.Context, args []string) error {
 	return c.Process(ctx)
 }
 
+func (c *PlanCommand) resolveStorageClient(ctx context.Context) (storage.Storage, error) {
```

Comment:
@drevell Added example usage here, I believe this is better because we have to parse the flagStorage, which isnt part of the storage implementation. We also generate different prefixes, based on if were local, or github, or whatever. The prefix generation will be abstracted once github is also pulled out of this command.